glide.yaml

```diff
@@ -29,29 +29,89 @@ import:
   version: ^2.2.6
   subpackages:
   - patricia
-- package: k8s.io/apimachinery
-  subpackages:
-  - pkg/api/errors
-  - pkg/api/meta
-  - pkg/apis/meta/v1
-  - pkg/fields
-  - pkg/runtime
-  - pkg/runtime/schema
-  - pkg/runtime/serializer
-  - pkg/util/intstr
-  - pkg/util/wait
-  - pkg/watch
 - package: k8s.io/client-go
-  version: ^4.0.0
+  version: d92e8497f71b7b4e0494e5bd204b48d34bd6f254
```

Comment:
why don't we pin it to a release? instead of a hash

Reply:
oh darnit.  good point